app/models/subscription.rb

```diff
@@ -8,4 +8,5 @@ class Subscription < ActiveRecord::Base
   after_create :send_new_message_notification # PushUpdatesForSubscription
 
   scope :unread, -> { where(state: 'unread') }
+  scope :of_active_user, -> { where(user_id: User.current_user.id) }
```

Comment:
If you pass in the User.current_user.id as a param then it's a useful scope for elsewhere.

```suggestion
  scope :for_user(user_id), -> { where(user_id: user_id) }
```